packages/api/src/router/stripe/index.ts

```diff
@@ -0,0 +1,141 @@
+import { z } from "zod";
+
+import { eq } from "@openstatus/db";
+import { user, usersToWorkspaces, workspace } from "@openstatus/db/src/schema";
+
+import { createTRPCRouter, protectedProcedure } from "../../trpc";
+import { stripe } from "./shared";
+import { webhookRouter } from "./webhook";
+
+const url =
+  process.env.NODE_ENV === "production"
+    ? "https://www.openstatus.dev"
+    : "http://localhost:3000";
+
+export const stripeRouter = createTRPCRouter({
+  webhooks: webhookRouter,
+
+  getUserCustomerPortal: protectedProcedure
+    .input(z.object({ workspaceId: z.string() }))
+    .mutation(async (opts) => {
+      const result = await opts.ctx.db
+        .select()
+        .from(workspace)
+        .where(eq(workspace.slug, opts.input.workspaceId))
```

Comment:
It appears to be here regarding the `slug` and `id`. Please use one or the other. I guess to be more consistent, we should stick with `slug`.